pkg/cnservice/server.go

```diff
@@ -117,6 +134,7 @@ func (s *service) initMOServer(ctx context.Context, pu *config.ParameterUnit) er
 func (s *service) initEngine(ctx context.Context, pu *config.ParameterUnit) error {
 	//TODO: initialize the engine
 	pu.StorageEngine = nil
```

Comment:
How do the cn-service initialize the engine.Engine and the TxnClient.

Reply:
added